### ISSUE TITLE
Failed to render Hl. 

### ISSUE THREAD

**Issue by polikesen123** [getinsomnia/insomnia#6862]
### Expected Behavior

<img width="1431" alt="image" src="https://github.com/Kong/insomnia/assets/52644550/7494a28d-06ed-442d-af78-3e04af9db497">

### Actual Behavior

Render Failure: e.includes is not a function
<img width="1434" alt="image" src="https://github.com/Kong/insomnia/assets/52644550/a8a5e115-148a-4043-94ea-6622ca0b8996">

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

Version: Insomnia 8.4.4 Build date: 11/23/2023 OS: Darwin arm64 23.0.0 Electron: 27.0.3 Node: 18.17.1 Node ABI: 118 V8: 11.8.172.16-electron.0 Architecture: arm64

### What operating system are you using?

macOS

### Operating System Version

macOS Sonoma 14.0

### Installation method

download from insomnia

**Comment by shanehandley** [comment]
Can also confirm this is happening on macOS Ventura 13.1 - noticed after update a few hours ago:

```
Version: Insomnia 8.4.4
Build date: 11/23/2023
OS: Darwin arm64 22.2.0
Electron: 27.0.3
Node: 18.17.1
Node ABI: 118
V8: 11.8.172.16-electron.0
Architecture: arm64
```

**Comment by vtka** [comment]
Same here:
```
TypeError: e.includes is not a function
    at KVe (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:470:868)
    at _l (file:///Applications/Insomnia.app/Contents/Resources/app.asar/debug-672b1df6.js:53:34237)
    at FS (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:42:19553)
    at aF (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:44101)
    at tF (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39819)
    at qpe (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39745)
    at Fd (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39596)
    at Px (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:35959)
    at QO (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:34906)
    at N (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:29:1615)
    at Immediate.me [as _onImmediate] (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:29:1987)
    at process.processImmediate (node:internal/timers:476:21)

**Comment by lchen4atlassian** [comment]
I have the same error after upgrading.

**Comment by DevJoseManuel** [comment]
Same here after upgrading:

```
TypeError: e.includes is not a function
    at KVe (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:470:868)
    at _l (file:///Applications/Insomnia.app/Contents/Resources/app.asar/debug-672b1df6.js:53:34237)
    at FS (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:42:19553)
    at aF (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:44101)
    at tF (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39819)
    at qpe (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39745)
    at Fd (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39596)
    at Px (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:35959)
    at QO (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:34906)
    at N (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:29:1615)
    at Immediate.me [as _onImmediate] (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:29:1987)
    at process.processImmediate (node:internal/timers:476:21)
```

**Comment by naimmalek** [comment]
same error here

**Comment by ihexxa** [comment]
Duplicated with: [https://github.com/Kong/insomnia/issues/6858](6858)
And please fallback to the previous version and we are actively working on this, sry for the inconvenience.

**Comment by kcsangit49** [comment]
updated and got this issue  on mac m1 

<img width="1278" alt="Screenshot 2023-11-23 at 10 34 56" src="https://github.com/Kong/insomnia/assets/148439415/299185d7-2285-40d1-9235-57877789eb3b">

**Comment by ThangarajAlagarsamy** [comment]
`TypeError: e.includes is not a function
    at KVe (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:470:868)
    at _l (file:///Applications/Insomnia.app/Contents/Resources/app.asar/debug-672b1df6.js:53:34237)
    at FS (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:42:19553)
    at aF (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:44101)
    at tF (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39819)
    at qpe (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39745)
    at Fd (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39596)
    at Px (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:35959)
    at QO (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:34906)
    at N (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:29:1615)
    at Immediate.me [as _onImmediate] (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:29:1987)
    at process.processImmediate (node:internal/timers:476:21)`

**Comment by dopla-dev** [comment]
Same here after the latest update: 

TypeError: e.includes is not a function
    at qVe (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:470:868)
    at _l (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/debug-1d1cf6bf.js:53:34237)
    at FS (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:42:19553)
    at oF (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:44101)
    at rF (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:39819)
    at Gpe (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:39745)
    at Bd (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:39596)
    at Px (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:35959)
    at eF (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:34906)
    at N (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:29:1615)
    at Immediate.me [as _onImmediate] (file:///C:/Users/**/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:29:1987)
    at process.processImmediate (node:internal/timers:476:21)

**Comment by korabh** [comment]
same here

**Comment by mrbeardy** [comment]
For some more context in case it's relevant, I'm getting this on a GQL request only, existing HTTP requests work fine.

Exporting the failing workspace and clearing the `text` body from the GQL request allows you to re-import it and not get the error, so seems like it may be related to that. 

Screenshot of the differing parts of the broken GQL request (top) and the re-imported and re-added GQL request (bottom):

<img width="821" alt="Screenshot 2023-11-23 at 9 57 31 am" src="https://github.com/Kong/insomnia/assets/578902/a90f800a-063d-445f-9a4b-648487a72b1a">

Everything else is the same.

I'm happy to send over the broken import file.

**Comment by bramstroker** [comment]
Same for me, broken for all GraphQL requests.

**Comment by Deuh** [comment]
Mentioned in other duplicate issue: 6863, 6858
- https://github.com/Kong/insomnia/issues/6863#issuecomment-1824340534
- https://github.com/Kong/insomnia/issues/6858#issuecomment-1823483381

Downgrade to this version 4.4.2 : https://github.com/Kong/insomnia/releases/tag/core%408.4.2 and it works

**Comment by Nigel-LJ-Fawcett** [comment]
Same problem here. Version 8.4.3 working immediately prior to the update. MacOS Sonoma 14.1.1 M1 arm64 chipset.

**Comment by filfreire** [comment]
Hey folks, we will launch a new release with a fix for this today, 8.4.5.

Closing this as a duplicate of #6857

**Comment by filfreire** [comment]
Update [here](https://github.com/Kong/insomnia/issues/6857#issuecomment-1824752852)
